configure.ac

```diff
@@ -4419,6 +4419,10 @@ fi
       fi
     fi
 
+    if test "x$mono_feature_disable_full_messages" != "xyes"; then
+      echo "HAVE_FULL_MESSAGES=1" >> $srcdir/$mcsdir/build/config.make
```

Comment:
I don't like this. I think it's overkill and it should be build profile driven for example we can completely ifdef it out for MOBILE